src/bin/nps.js

```diff
@@ -102,6 +102,20 @@ function onInit() {
   ))
 }
 
+function getConfigType() {
+  let configType
```

Comment:
I think this could be written a little more simply:

```javascript
return includes(process.argv, '--type') ? process.argv[indexOf(process.argv, '--type') + 1] : 'js'
```

Though... I'm not sure that's actually _simpler_... Just more terse. What do you think?